src/declarations.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'mathjax-node' {
```

Comment:
It doesn't look as if this file is being picked up by the compilation running via `npm generate:mathjax`, otherwise I'd expect https://travis-ci.org/stencila/thema/builds/650448345 to be green (although I may have missed something). Might be something to do with using `ts-node` rather than `tsc`, as a locally running a `tsc` compilation over `mathjax.ts` is clean.

Reply:
ah, indeed that seems to be the case.
To fix we need to let `ts-node` know to read the project files by setting the `--files` flag.
I checked out your branch and running `npx ts-node --files src/generate/mathjax.ts` compiled successfully.

Reply:
This has now been done